### ISSUE TITLE
Review 'small GTPase mediated signal transduction' and children

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20864]
While working on #19082 we noticed other terms relevant to small GTPases - we wonder whether those are realy signaling pathways or should be incorporated into other BPs 

'small GTPase mediated signal transduction'
'Ras protein signal transduction'
'ARF protein signal transduction'
'Rab protein signal transduction'
'Rac protein signal transduction'
'Ral protein signal transduction'
'Ran protein signal transduction'
'Rap protein signal transduction'
'Rho protein signal transduction'

For example
- 'Ran protein signal transduction' -> nucleocytoplasmic transport
- 'Rab protein signal transduction' -> intracellular trafficking

**Comment by ValWood** [comment]
I agree, and I have almost removed. We either know the actual process or pathway, or it's just a molecular function.

**Comment by pfey03** [comment]
I had missed this ticket and we have 19 Dicty annotations to children of this term. I will have to inspect them all.

**Comment by pfey03** [comment]
Also, what about the regulation terms. Those are not among the 19 but we have those too. Do those also have to go?
